### PR TITLE
Fixed footer space when customized

### PR DESCRIPTION
**Pull request by Godmartinz**
# Description
Footer would be covered up by the div wrapper if the footer went more than one line. I added display:grid to the footer to stack the support info, the Snipe-it Link and the footer. 
I created another css class `pull-text-right` because the `pull-right` was not working in this case. The footer dynamically grows with the size of a custom footer message

BEFORE:
image

AFTER:
image

Fixes #SC-15580

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* PHP version:
* MySQL version
* Webserver version
* OS version

# Checklist:

- [ ] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [ ] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by snipe — CHANGES_REQUESTED** [review]
This still seems a little brittle to me. I'd think it shouldn't need to be hard-coded, but would instead increase as the number of lines increase, especially since that section takes some amount of markdown, IIRC

**Comment by Godmartinz** [comment]
Ok, I'll see what I can conjure up 👍

**Comment by Godmartinz** [comment]
Went with display:flex and now it dynamically adjusts with the size of the footer content or media size.

**Comment by Godmartinz** [comment]
shoot. 1 sec found a visual bug 🙁

**Comment by Godmartinz** [comment]
Ok the support footer text aligns to the right, as it should.

File: resources/views/layouts/default.blade.php [hunk 1]
```diff
@@ -850,10 +850,12 @@ class="fas fa-check text-blue fa-fw"></i>
...
-
-            <footer class="main-footer hidden-print">
-
-                <div class="pull-right hidden-xs">
+            @if ($snipeSettings->footer_text!='')
+                <footer class="main-footer hidden-print" style="display:grid;flex-direction:row;" >
+            @else
+                <footer class="main-footer hidden-print">
+            @endif
+                <div class="pull-text-right hidden-xs">
```
Comment (snipe) [inline resources/views/layouts/default.blade.php:858]: Do we want to actually hide this on `xs` breakpoints?

**Comment by snipe** [comment]
This is already merged, but I may have to revert it. It's making the spacing at the bottom way too big. See SC-23134

**Pull request merged**